packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -619,12 +619,6 @@ async function handler(req: CustomRequest) {
     return { message: "Booking successfully cancelled." };
   }
 
-  const attendeeDeletes = prisma.attendee.deleteMany({
```

Comment:
@zomars @emrysal Why were we deleting the attendees? There should be some reason for it.